### PR TITLE
Separate Hide Typing and Hide Online options

### PR DESCRIPTION
**Pull request by pablo-gbr**
When hiding the Online status, you won't get any updates from WhatsApp, so, if you want to hide just the 'Typing...' status, you will also hide all the online and last seen statuses from your contacts.

This separates the two options, making you able to hide the typing status but remaining online.

I tested it and it seemed to work. Feel free to test it and let me know any issues :)

### CODE REVIEW

File: core/ui.js [hunk 1]
```diff
@@ -213,7 +215,8 @@ async function addIconIfNeeded()
 
 function generateDropContent(options)
 {
-    var presenceCaption = "Will prevent you from seeing other people's last seen and typing status";
+    var onlineStatusCaption = "Will prevent you from being online. If you enable this, you won't be able to see others' status.";
```
Comment (tomer8007) [inline core/ui.js:218]: "Prevent you from being online" sounds a bit weird. Maybe we can write "Stops sending presence updates. If enabled, will prevent you from seeing other's online status"

File: core/ui.js [hunk 2]
```diff
@@ -268,30 +274,38 @@ function generateDropContent(options)
...
-                    <div id='incognito-option-presence-updates' class='incognito-options-item' style='cursor: pointer;'>
+                    <div id='incognito-option-online-status' class='incognito-options-item' style='cursor: pointer;'>
                         <div class='checkbox-container-incognito' style=''>
-                            <div class='checkbox checkbox checkbox-incognito ${presenceUpdatesChcekbox}
+                            <div class='checkbox checkbox checkbox-incognito ${onlineUpdatesCheckbox}
                             </div>
                         </div>
-                        Don't send \"Last Seen\" and \"Typing\" updates
-                        <div class='incognito-options-description'>${presenceCaption}</div>
+                        Hide \"Online\" status
```
Comment (tomer8007) [inline core/ui.js:282]: Let's have the titles as variables to make it consistent with the other titles

**Comment by tomer8007** [comment]
Thank you. Can you please attach a screenshot of how the GUI looks like?

File: core/ui.js (hunk 2 above)
Comment (pablo-gbr) [reply core/ui.js]: Done! I added the read confirmations title to the variables list, they were missing too.

**Comment by pablo-gbr** [comment]
> Thank you. Can you please attach a screenshot of how the GUI looks like?

Sure, it looks like this on my end
image
image
image

File: core/ui.js [hunk 3]
```diff
@@ -213,7 +215,16 @@ async function addIconIfNeeded()
 
 function generateDropContent(options)
 {
-    var presenceCaption = "Will prevent you from seeing other people's last seen and typing status";
+    var onlineStatusTitle = "Hide \"Online\" status";
+    var onlineStatusCaption = "Stops sending presence updates. If enabled, will prevent you from seeing others' online status.";
+
+    var typingStatusTitle = "Hide \"Typing...\" status";
+    var typingStatusCaption = "Will prevent you from sending 'Typing...' status.";
```
Comment (tomer8007) [inline core/ui.js:222]: Actually the "will you prevent you" here also sounds weird. Let's just say "Stops sending composing updates"

Comment (pablo-gbr) [reply core/ui.js]: Done

**Pull request merged**